kubernetes/istio_details_service.go

```diff
@@ -219,6 +219,25 @@ func (in *IstioClient) GetDestinationRules(namespace string, serviceName string)
 	return destinationRules, nil
 }
 
+func (in *IstioClient) GetAllDestinationRules() ([]IstioObject, error) {
+	result, err := in.istioNetworkingApi.Get().Resource(destinationRules).Do().Get()
```

Comment:
@xeviknal this is not going to work in a protected environment.
You are using here the syntax "FETCH ALL" from k8s, at the moment that user doesn't have a namespace right, this query is going to return an error.

So, this has to be done in a different way, it should first query all namespaces that user has access and then fetch objects one by one.

Reply:
I have approached this too. I fetching all the DestinationRules attached to all the namespaces the user has access to. Thanks for noticing this. Wasn't aware of that issue.